includes/admin/class-amp-post-meta-box.php

```diff
@@ -213,6 +213,27 @@ public function enqueue_block_assets() {
 			return;
 		}
 
+		// Gutenberg v5.4 was bundled with WP 5.2, which is the earliest release known to work without errors.
+		$gb_supported = defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '5.4.0', '>=' );
+		$wp_supported = ! $gb_supported && version_compare( get_bloginfo( 'version' ), '5.2', '>=' );
+
+		// Let the user know that block editor functionality is not available if the current Gutenberg or WordPress version is not supported.
+		if ( ! $gb_supported && ! $wp_supported ) {
+			wp_add_inline_script(
+				'wp-edit-post',
+				sprintf(
+					'wp.domReady(
+						function () {
+							wp.data.dispatch( "core/notices" ).createWarningNotice( %s )
+						}
+					);',
+					wp_json_encode( __( 'AMP functionality is not available in the Block Editor as the current version installed is not supported. Please update the Block Editor plugin to the latest version or use the Classic Editor instead.', 'amp' ) )
```

Comment:
```suggestion
					wp_json_encode( __( 'AMP functionality is not available since your version of the Block Editor is too old. Please either update WordPress core to the latest version or activate the Gutenberg plugin. As a last resort, you may use the Classic Editor plugin instead.', 'amp' ) )
```